src/Orleans/AzureUtils/AzureTableDataManager.cs

```diff
@@ -152,7 +150,7 @@ public AzureTableDataManager(string tableName, string storageConnectionString, T
         /// </summary>
         /// <param name="data">Data to be inserted into the table.</param>
         /// <returns>Value promise with new Etag for this data entry after completing this storage operation.</returns>
-        public async Task<string> CreateTableEntryAsync(T data)
+        public async Task<string> TryCreateTableEntryAsync(T data)
```

Comment:
Isn't the pattern with Try* to return a bool? I think this one would be a bit misleading as it returns a Task<string> and there's no way to do write 

```csharp
if( TryCreateTableEntryAsync(..))
```